guides/model-state.md

```diff
@@ -0,0 +1,13 @@
+## Avoid Local/UI state on Models
+
+Models live on the `store` service, and all services hold "application state."
+UI or local state should not be entangled with application state by adding
+it to a model.
```

Comment:
I'm not sure I agree with this statement in 100% of use cases. I often use https://github.com/stefanpenner/ember-state-services to hold UI state, particularly in cases where the lifetime of the state is different than the lifetime of the component used to visualize that UI state.